pkgs/games/space-cadet-pinball/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub, fetchzip
+, cmake, SDL2, SDL2_mixer
+, unrar-wrapper, makeWrapper
+}:
+
+let
+  assets = (fetchzip {
+    url = "https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar";
+    sha256 = "sha256-fC+zsR8BY6vXpUkVd6i1jF0IZZxVKVvNi6VWCKT+pA4=";
+    stripRoot = false;
+  }).overrideAttrs (old: {
+    nativeBuildInputs = old.nativeBuildInputs ++ [ unrar-wrapper ];
+  });
+in
+stdenv.mkDerivation rec {
+  pname = "SpaceCadetPinball";
+  version = "389122182e34f9b37d5476b797bb7b7b4d217f51";
+
+  src = fetchFromGitHub {
+    owner = "k4zmu2a";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Ieou3ScDlcTu3bIiKwY6j2cnwldcI3DBTROf1zHXBWw=";
+  };
+
+  buildInputs = [
+    SDL2
+    SDL2_mixer
+    cmake
+    makeWrapper
+  ];
+
+  postInstall = ''
+    mkdir -p $out/lib/${pname}
+    mv $out/bin/${pname} $out/lib/${pname}
+    install ${assets}/*.{DAT,DOC,MID,BMP,INF} ${assets}/Sounds/*.WAV $out/lib/${pname}
+
+    makeWrapper $out/lib/${pname}/${pname} $out/bin/${pname} --argv0 ${pname}
+  '';
+
+  meta = with lib; {
+    description = "Reverse engineering of 3D Pinball for Windows – Space Cadet, a game bundled with Window";
+    homepage = "https://github.com/k4zmu2a/SpaceCadetPinball";
+    license = licenses.unfree;
```

Comment:
Just to understand, the code itself is MIT however the issue is the assets right?

Reply:
I think it would be nice to have a comment clarifying this.

Reply:
> Just to understand, the code itself is MIT, however the issue is the assets right?

The assets are surely unfree. However the code itself, despite being labeled as MIT in the repo, is in a grey area (see https://github.com/k4zmu2a/SpaceCadetPinball/issues/1).

Reply:
Also, the (https://aur.archlinux.org/packages/spacecadetpinball-git/)[package in the AUR] has it labeled as both proprietary and MIT. I like this approach